astroquery/jplhorizons/core.py

```diff
@@ -35,7 +36,7 @@ class HorizonsClass(BaseQuery):
     TIMEOUT = conf.timeout
 
     def __init__(self, id=None, location=None, epochs=None,
-                 id_type='smallbody'):
+                 id_type=None):
```

Comment:
It doesn't feel right to have a default that raises a warning. 

Maybe keep `None` as one that falls back whatever the Horizon default is (allowing it to change in the future), or have `''` as out default to mirror Horizon? I don't have an option of which way to go, besides to not have a warning for whatever we choose.

Reply:
OK, so we can keep `None` in perpetuity.  But no warning that we will make a breaking change to the defaults?

Reply:
It's in the changelog 🤷 

However, in practice, do we expect any practical differences? E.g. is there any overlap between major and small bodies? Or maybe searches will take a tiny bit longer?

Reply:
Unfortunately, there is overlap.  A search for the asteroid Io would succeed under the current astroquery default, but Horizons's default behavior would instead match the Jovian satellite Io, and not even mention the asteroid.  And based on our discussion in #1742, we want astroquery to be better aligned with Horizons's behavior.  For example,
```
In [3]: Horizons('Metis').ephemerides()                                         
Out[3]: 
<Table masked=True length=1>
    targetname          datetime_str       ...  PABLon   PABLat
       ---                  ---            ...   deg      deg  
      str17                str24           ... float64  float64
----------------- ------------------------ ... -------- -------
9 Metis (A848 HA) 2021-Sep-28 17:13:01.870 ... 228.1162  1.2427

In [4]: Horizons('Metis', id_type='majorbody').ephemerides()                    
Out[4]: 
<Table masked=True length=1>
 targetname       datetime_str          datetime_jd    ...  PABLon   PABLat
    ---               ---                    d         ...   deg      deg  
   str11             str24                float64      ... float64  float64
----------- ------------------------ ----------------- ... -------- -------
Metis (516) 2021-Sep-28 17:13:15.853 2459486.217544595 ... 326.5604 -1.0923

```
Note the different target names.

Reply:
ah, ok, I didn't know satellites are major. Still I feel the changelog should be enough. Maybe we can ask Adam to tweet highlight it at release time, too.

(or mention  it in  a "note" or "warning" in the narrative docs)

Reply:
OK, I'll remove the warnings, and your comment reminds me to review the documentation, too!

Reply:
Done.